### PR TITLE
Fix compatibility issues with 2.1.0-dev.9.4.flutter-f9ebf21297 stable…

### PR DESCRIPTION
**Pull request by Matcho0**
… flutter SDK

Using the stable SDK branch and Android Studio for Flutter development, the app_review plugin would not build due to the following errors:

Get packages failed due to pubspec.yaml:

The current Dart SDK version is 2.1.0-dev.9.4.flutter-f9ebf21297.

Because myApp depends on app_review from path which requires SDK version >=2.1.0 <3.0.0, version solving failed.

After resolving that, build.gradle caused the following error when attempting issuing 'flutter run':

* What went wrong:
Execution failed for task ':app:preDebugBuild'.
> Android dependency 'com.android.support:support-media-compat' has different version for the compile (26.1.0) and runtime (27.1.0) classpath. You should manually set the same version via DependencyResolution

The changes to pubspec.yaml and build.gradle in this PR resolve these issues.

### CODE REVIEW

File: packages/app_review/pubspec.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ author: Rody Davis <rody.davis.jr@gmail.com>
 homepage: https://github.com/AppleEducate/plugins
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.0.0-dev.68.0 <3.0.0"
```
Comment (sjmcdowall) [inline packages/app_review/pubspec.yaml:8]: FWIW -- I don't think this is the right change.. you are going back to 2.0.0 and it should be a 2.1.0+ or something... 

I think you mean:

sdk: '>=2.1.0-dev.68.0 <3.0.0'

**Review by sjmcdowall — CHANGES_REQUESTED** [review]

File: packages/app_review/pubspec.yaml (hunk 1 above)
Comment (Matcho0) [reply packages/app_review/pubspec.yaml:8]: This version is directly from the example project Android Studio creates and is the version that most other compatible plugins use. >=2.0.0-dev.68.0 <3.0.0 simply means an SDK between those two values, and not necessarily going back to 2.0.0.  It's better for backwards compatibility and allows for this plugin's usage across more SDKs

Comment (sjmcdowall) [reply packages/app_review/pubspec.yaml:8]: OK -- as long as it works -- but I think many many plugins now demand SDK >= 2.1.XX and 2.0 will be flagged as an error if they stay there.. <shrug>

Comment (rodydavis) [reply packages/app_review/pubspec.yaml:8]: I originally had it for 2.1.0 but changed it as there are a lot of plugins that haven't updated yet and some of my plugins depend on other plugins. Once they update I will too.

**Comment by rodydavis** [comment]
Is this still happening?

This cannot be merged until the conflicts are fixed. Thank you for the PR!

**Review by rodydavis — APPROVED** [review]
Looks great!

**Comment by sjmcdowall** [comment]
As of THIS morning -- this should no longer be an issue since dart is now a full 2.1.2 and thus no more odd semvar issues for >= 2.1.0

**Comment by rodydavis** [comment]
Ok perfect!

**Pull request merged**